pkg/action/install.go

```diff
@@ -643,6 +643,11 @@ func (c *ChartPathOptions) LocateChart(name string, settings *cli.EnvSettings) (
 				return "", err
 			}
 		}
+		if c.RepoURL != "" {
+			warning("Chart found at local path '%v'. Ignoring Repository URL '%v.'",
```

Comment:
In `action.Configuration`, we have the log field. It might be possible to use it to output. WDYT

Reply:
Thanks.

There's a log field in:
~~~
 69 type Install struct {
 70         cfg *Configuration
~~~

That's a private attribute. I could access that with a Getter which I'd have to create (`action.Install.GetLog()`) and then do:
~~~
172 func runInstall(args []string, client *action.Install, valueOpts *values.Options, out io.Writer) (*release.Releas    e, error) {
(...)
185         cp, err := client.ChartPathOptions.LocateChart(chart, settings, client.GetLogger())
(...)
~~~

And I'd have to change that everywhere where I encounter `LocateChart`.

The problem then is that I'd have to change that for action.Show, as well, but action.show does not have a Configuration object:
~~~
cmd/helm/show.go:179:74: client.GetLog undefined (type *action.Show has no field or method GetLog)
~~~

Reply:
> Thanks.
> 
> There's a log field in:
> 
> ```
>  69 type Install struct {
>  70         cfg *Configuration
> ```
> 
> That's a private attribute. I could access that with a Getter which I'd have to create (`action.Install.GetLog()`) and then do:
> 
> ```
> 172 func runInstall(args []string, client *action.Install, valueOpts *values.Options, out io.Writer) (*release.Releas    e, error) {
> (...)
> 185         cp, err := client.ChartPathOptions.LocateChart(chart, settings, client.GetLogger())
> (...)
> ```
> 
> And I'd have to change that everywhere where I encounter `LocateChart`.
> 
> The problem then is that I'd have to change that for action.Show, as well, but action.show does not have a Configuration object:
> 
> ```
> cmd/helm/show.go:179:74: client.GetLog undefined (type *action.Show has no field or method GetLog)
> ```

Yes, so I think the warning should be thrown to the structure that references it to handle
It should not be handled in LocateChart